packages/openapi3-parser/lib/parser/parseMap.js

```diff
@@ -15,8 +17,8 @@ const parseMapMember = R.curry((context, parser, member) => {
   const Member = R.constructN(2, context.namespace.elements.Member)(member.key);
 
   const parseResult = R.map(
-    R.unless(isAnnotation, Member),
-    parser(context, member.value)
+    R.unless(isAnnotationOrMember, Member),
```

Comment:
I'm not going to block on it, however it might be a good idea to have coverage for this particular change in `parseMap-test`. I know it is tested through integration, although I've found that when changing (and breaking) these types of reusable components where behaviour is used upon many other places. It is a lot harder to figure out the case through debugging the integration.

Reply:
A test for this was added in https://github.com/apiaryio/api-elements.js/pull/528/commits/6f9ae7e49bf659a88a7ce3aa9998aabc2059f257